problems/low-bus-factor.md

```diff
@@ -0,0 +1,88 @@
+---
+slug: low-bus-factor
+stages:
+  - development
+short_description: Low bus factor is a risk for team work when one or more important team members get stuck (from various reasons) and cannot work. 
+tags:
+  - Low bus factor
+  - team
+  - redundancy
+  - fresh documentation
+  - automation
+keywords:
+  - bus
+  - factor
+  - team
+  - methodology
+---
+
+# Low Bus Factor
+
+## TL;DR
+
+How many people need to get hit by a bus for the project or team to stop functioning? What are the ways how to increase this number?  
+
+![](/files/bus1.png)
+
+## What Is Bus Factor
+
+The “bus factor” is the minimum number of team members that have to disappear from a project suddenly before the project stalls due to lack of knowledgeable or competent personnel. 
+
+**The lower the number, the higher the risks.**
+
+## How to Increase Bus Factor
+
+### Problem With Rockstars
+
+Rockstars  – *Mr/Ms-know-it-all* – are nice to have in a team. But, what if he/she gets hit by a bus (or gets sick, headhunted, etc.)? In fact, rockstars usually are not the problem. The problem is that many teams rely too much on their rockstars and forget about the impact of losing them. 
+
+Try to observe what happens when your rockstar is out of office. If everything slows down, then you’re probably in trouble. The next sections explain ways how to increase your bus factor, regardless you have rockstars on your team or not.
+
+### Redundancy
+
+To make sure that your team continues to function if a key member is unavailable, you need to ensure that such a person has at least one *backup*. These *backups* should be able to take over any critical role (function, task) of the unavailable team member at any time. 
```

Comment:
If it makes sense to you, you could link to the article about responsibility here https://developerexperience.io/practices/responsibility

Reply:
Added